src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -0,0 +1,272 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ReactNode } from "react";
+import Image from "next/image";
+import Script from "next/script";
+
+import "../../../../client/css/index.css";
+import AppConstants from "../../../../appConstants.js";
+import MonitorLogo from "../../../../client/images/monitor-logo-transparent@2x.webp";
+import MozillaLogo from "../../../../client/images/moz-logo-1color-white-rgb-01.svg";
+import { getL10n } from "../../../functions/server/l10n";
+import { getCurrentSession } from "../../../functions/server/sessionHelpers";
+
+export type Props = {
+  children: ReactNode;
+};
+
+const dummyData = {
+  partial: {
+    name: "breaches",
+  },
+  meta: {
+    title: "",
+    socialTitle: "",
+    socialDescription: "",
+  },
+  pathname: "/",
+  fxaProfile: {
+    avatar: "https://profile.accounts.firefox.com/v1/avatar/f",
+  },
+};
+
+import RelayLogo from "../../../../client/images/logo-relay.svg";
+import VPNLogo from "../../../../client/images/logo-vpn.svg";
+import OpenInIcon from "../../../../client/images/icon-open-in.svg";
+import SettingsIcon from "../../../../client/images/icon-settings.svg";
+import HelpIcon from "../../../../client/images/icon-help.svg";
+import SignOutIcon from "../../../../client/images/icon-signout.svg";
+
+const MainLayout = async (props: Props) => {
+  const session = await getCurrentSession();
+  const l10n = getL10n();
+
+  const isBreachesPage = dummyData.partial.name === "breaches";
+  const isSettingsPage = dummyData.partial.name === "settings";
+
+  const UserMenu = ({ userData }) => {
+    const fxaUser = userData?.fxa_profile_json;
+
+    return (
+      <div className="user-menu-wrapper" tabIndex={-1}>
+        <Script src="/nextjs_migration/client/js/userMenu.js" />
+        <button
+          aria-expanded="false"
+          aria-haspopup="true"
+          className="user-menu-button"
+          title={l10n.getString("menu-button-title")}
+        >
+          <img src={fxaUser?.avatar} alt={l10n.getString("menu-button-alt")} />
+        </button>
+        <menu
+          aria-label={l10n.getString("menu-list-accessible-label")}
+          className="user-menu-container user-menu-popover"
+          role="navigation"
+          hidden
+        >
+          <li tabIndex={1}>
+            <a
+              href={AppConstants.FXA_SETTINGS_URL}
+              target="_blank"
+              className="user-menu-header"
+            >
+              <b className="user-menu-email">{fxaUser.email}</b>
+              <div className="user-menu-subtitle">
+                {l10n.getString("menu-item-fxa")}
+                <Image alt="" src={OpenInIcon} />
+              </div>
+            </a>
+          </li>
+          <hr />
```

Comment:
Oh. Thanks for catching this one. This right now is how we have it on the existing Monitor site — I’ll replace it because invalid syntax is really not optimal. Otherwise, the JSX for the pages is almost one-to-one taken from the existing string literals as a first step to replace the existing architecture with Next.